app/assets/src/components/visualizations/table/base_table.scss

```diff
@@ -33,6 +34,15 @@
   outline: 0;
   text-align: center;
   text-transform: none;
+
+  .text {
+    display: inline-block;
+    max-width: 100%;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    margin-top: 7px;
```

Comment:
Did you just copy from current style?
I wonder if we should vertically align it.

Reply:
yes those lines were here https://github.com/bvaughn/react-virtualized/blob/8738097a6ead422f74590d6a631bd5fd83ebc831/source/styles.css#L38 besides the margin-top. Switched to the vertical-align and it looks the same 👍